File: Team5/Akanksha_Singh/Week-1/MoveZeroes.cpp
class Solution {
public:
    
    void moveZeroes(vector<int>& nums) {
        int j=0;
        int count=0;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]!=0)
            {
                nums[j]=nums[i];
                j++;
                count++;
            }
        }
        for(int i=count;i<nums.size();i++)
        {
            nums[i]=0;
        }
    }
};

File: Team1/Riya Tanwar/move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int n = nums.length;
        int count = 0; 
        for (int i = 0; i < n; i++) {
            if (nums[i] != 0) {
                if (count != i) { 
                    nums[count] = nums[i];
                    nums[i] = 0;
                }
                count++;
            }
        }
    
    }
    }